examples/runRieszWaveletAnalysis.cpp

```diff
@@ -77,7 +77,7 @@ runRieszWaveletPhaseAnalysis( const std::string& inputImage,
   using FFTPadFilterType = itk::FFTPadImageFilter<ImageType>;
   auto fftPadFilter = FFTPadFilterType::New();
   fftPadFilter->SetInput(reader->GetOutput());
-  fftPadFilter->SetBoundaryConditionToConstant(0);
+  fftPadFilter->SetBoundaryCondition(0);
```

Comment:
That means no boundary conditions. I think the right way to set the boundaries is with something like:

```cpp
using BoundaryConditionType = itk::ConstantBoundaryCondition<ImageType>;
BoundaryConditionType bounds;
bounds.SetConstant(itk::NumericTraits<typename ImageType::PixelType>::ZeroValue());
fftPadFilter->SetBoundaryCondition(&bounds);

```

Reply:
The `SetBoundaryConditionToConstant` was a bug, from using another type of FFT Pad ImageFilter in the past! Thanks for spotting it.

Reply:
Yes I felt it is not the right way, but left it with 0 as a placeholder. Thanks for correcting !